src/components/charts/Vis5.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react'
+import { Scatter } from 'react-chartjs-2'
+import DATA from './vis5.json'
+import zoomPlugin from 'chartjs-plugin-zoom'
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js'
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  zoomPlugin
+)
+
+export const options = {
+  type: 'line',
+  responsive: true,
+  parsing: {
+    xAxisKey: 'Mean age of the air (yr BP)',
```

Comment:
Yep, you could try to use time scale https://www.chartjs.org/docs/latest/samples/scales/time-line.html but I don't know if it will work with years before present.
But if it is integer, then, probably it could be left as it is (remember that the bigger year, the older it is)
https://en.wikipedia.org/wiki/Before_Present